commands/ApiController.php

```diff
@@ -31,6 +31,11 @@ class ApiController extends BaseController
      */
     public $guidePrefix = 'guide-';
 
+    /**
+     * @var string URL for the README to use for the index of the guide.
+     */
+    public $readmeUrl = "https://raw.github.com/yiisoft/yii2-framework/master/README.md";
```

Comment:
Why URL instead of local file path?

Reply:
That URL was hard-coded in the bootstrap template which I copied.  For it to work without changing the command line params, it has to match the value that was in the previous template.